### PR TITLE
Adding IS31FL3741

### PR DESCRIPTION
**Pull request by kattni**

### CODE REVIEW

File: docs/drivers.rst [hunk 1]
```diff
@@ -287,6 +287,7 @@ Other
     Character LCD <https://circuitpython.readthedocs.io/projects/charlcd/en/latest/>
     HT16K33 LED Matrices and Segment Displays <https://circuitpython.readthedocs.io/projects/ht16k33/en/latest/>
     IS31FL3731 Charlieplexed LED Matrix <https://circuitpython.readthedocs.io/projects/is31fl3731/en/latest/>
+    IS31FL3741 RGB LED Matrix driver <https://circuitpython.readthedocs.io/projects/is31fl3741/en/latest/>
```
Comment (jerryneedell) [inline docs/drivers.rst:290]: Do both of these lines belong here? they point to the same URL?

Comment (dhalbert) [reply docs/drivers.rst:290]: They differ in one character:
circuitpython.readthedocs.io/projects/is31f**l3731**/en/latest/
circuitpython.readthedocs.io/projects/is31fl**3741**/en/latest/

**Review by dhalbert — APPROVED** [review]
:+1:

File: docs/drivers.rst (hunk 1 above)
Comment (kattni) [reply docs/drivers.rst:290]: One is 3731, the second is 3741. The new one is one digit off.

**Pull request merged**